### ISSUE TITLE
Bad formed .py file for ENTITY-MIB

### ISSUE THREAD

**Issue by jccardonar** [etingof/pysmi#20]
(Not sure if this issue should be filed here or under the pysnmp project)

I am experiencing problems when trying to use pysnmp with the ENTITY-MIB. It seems pysmi  is building an incorrect .py file for the ENTITY-MIB. 

To reproduce, make sure that the .py for ENTITY-MIB has not been generated before. Place the ENTITY-MIB, and its requirements in a folder. I downloaded all of them from http://mibs.snmplabs.com/asn1/:

```
wget http://mibs.snmplabs.com/asn1/ENTITY-MIB
wget http://mibs.snmplabs.com/asn1/SNMP-FRAMEWORK-MIB
wget http://mibs.snmplabs.com/asn1/SNMPv2-CONF
wget http://mibs.snmplabs.com/asn1/SNMPv2-SMI
wget http://mibs.snmplabs.com/asn1/SNMPv2-TC
```
Now, try to compile and load the mib:

```
from pysnmp.smi import builder, view, compiler
import pysnmp

mibBuilder = pysnmp.smi.builder.MibBuilder()
compiler.addMibCompiler(mibBuilder, sources=("/home",))
com = mibBuilder.getMibCompiler()
com.compile("ENTITY-MIB")
mibBuilder.loadModule("ENTITY-MIB")
```

Which, to me, raises an exception:

`MibLoadError: MIB module "/root/.pysnmp/mibs/ENTITY-MIB.py" load error: ['Traceback (most recent call last):\n', '  File "/usr/local/lib/python2.7/dist-packages/pysnmp/smi/builder.py", line 331, in loadModule\n    exec (modData, g)\n', '  File "<string>", line 127\n', "    entConfigChange = NotificationType((1, 3, 6, 1, 2, 1, 47, 2, 0, 1))if mibBuilder.loadTexts: entConfigChange.setStatus('current')\n",`

The reason for it is a missing break line before the if statement. Adding it manually solves the problem.

I hope you can reproduce the problem.

**Comment by etingof** [comment]
Thank you for the heads up!

Fixed in master, release is on its way.

**Comment by etingof** [comment]
Released pysmi 0.2.2 on PyPI at GitHub.

**Comment by jccardonar** [comment]
And build is back to normal. Thanks a lot @etingof
